README.md

```diff
@@ -121,37 +121,37 @@ liota.conf provides path to find out various configuration & log files. When ini
 * Looks in the current working directory '.'
 * User's home directory '~'
 * A LIOTA_CONF environment variable
-* Finally the default location for every installation: /etc/liota/conf.
+* Finally the default location for every installation: /etc/liota/. (note this will need to be copied from the system doc directory, typically /usr/share/doc/liota-&lt;version&gt;/config/)
 
 Here is the default liota.conf file:
 
 ```bash
 [LOG_CFG]
-json_path = /etc/liota/conf/logging.json
+json_path = /etc/liota/logging.json
 
 [LOG_PATH]
 log_path = /var/log/liota
 
 [UUID_PATH]
-uuid_path = /etc/liota/conf/uuid.ini
+uuid_path = /etc/liota/uuid.ini
 
 [IOTCC_PATH]
 dev_file_path = /etc/liota/conf/devs
```

Comment:
Will it be fine to change IoTCC Path as well from "/etc/liota/conf" to "/etc/liota" as by default syswide path is changed to  '/etc/liota/' in utility.py?